src/SFA.DAS.ApplyService.Application/Apply/IApplyRepository.cs

```diff
@@ -33,12 +41,12 @@ public interface IApplyRepository
         Task UpdateSections(List<ApplicationSection> sections);
         Task UpdateSequences(List<ApplicationSequence> sequences);
         Task SaveSection(ApplicationSection section, Guid? userId = null);
-        Task<List<Domain.Entities.Apply>> GetOpenApplications();
-        Task<List<Domain.Entities.Apply>> GetFeedbackAddedApplications();
-        Task<List<Domain.Entities.Apply>> GetClosedApplications();
-        Task<List<Domain.Entities.Apply>> GetOpenFinancialApplications();
-        Task<List<Domain.Entities.Apply>> GetFeedbackAddedFinancialApplications();
-        Task<List<Domain.Entities.Apply>> GetClosedFinancialApplications();        
+        Task<List<RoatpApplicationSummaryItem>> GetOpenApplications();
+        Task<List<RoatpApplicationSummaryItem>> GetFeedbackAddedApplications();
+        Task<List<RoatpApplicationSummaryItem>> GetClosedApplications();
+        Task<List<RoatpFinancialSummaryItem>> GetOpenFinancialApplications();
+        Task<List<RoatpFinancialSummaryItem>> GetFeedbackAddedFinancialApplications();
+        Task<List<RoatpFinancialSummaryItem>> GetClosedFinancialApplications();        
```

Comment:
Can these be moved above the comment saying '// NOTE: This is old stuff or things which are not migrated over yet'

Reply:
Done